runPatchwork.sh

```diff
@@ -1 +1 @@
-spark-submit --class PatchWorkDemo --master local[4] bin/patchwork_1.0.jar
+spark2-submit --class PatchWorkDemo --master local[4] bin/patchwork_1.1.jar
```

Comment:
The [`/bin` directory](https://github.com/crim-ca/patchwork/tree/MAJ-spark-2/bin) needs to be updated.
@Mignastor Could-you add a precompiled jar of the version 1.1 (and remove the 1.0 jar) ?